GenericBot/CommandModules/RoleCommands.cs

```diff
@@ -62,6 +62,7 @@ public List<Command> GetRoleCommands()
                 {
                     await msg.ReplyAsync(str);
                 }
+                message += $"\n _\\You can also use `{prefix}rolestore save` to backup your assigned roles";
```

Comment:
What is the formatting with the _\\ supposed to be? Looks incomplete

Reply:
was me being a silly >.> fixed now!